src/components/board/board.test.jsx

```diff
@@ -1,19 +1,21 @@
 import React from 'react';
-import Board from './board';
 import { shallow, mount } from 'enzyme';
+import Board from './board';
 
-it('<Board/> renders without error', () => {
-  let squares = Array(9).fill(null);
-  shallow(<Board squares={squares} />);
-});
+describe('Board', () => {
+  it('renders without error', () => {
+    const squares = Array(9).fill(null);
+    shallow(<Board squares={squares} />);
```

Comment:
A better test here would be to add an assertion to make sure that `<Board />` exists.

Reply:
Added assertion to check if component exists